recipes/cython_gsl/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set name = "CythonGSL" %}
+{% set name = "cython_gsl" %}
```

Comment:
I can change it to cythongsl no problem, I thought it might have to match the package name,
see https://github.com/twiecki/CythonGSL/blob/master/setup.py#L10 but you're saying it doesn't matter?